packages/aws-cdk/lib/cli.ts

```diff
@@ -255,7 +256,8 @@ async function parseCommandLineArguments(args: string[]) {
     .command('destroy [STACKS..]', 'Destroy the stack(s) named STACKS', (yargs: Argv) => yargs
       .option('all', { type: 'boolean', default: false, desc: 'Destroy all available stacks' })
       .option('exclusively', { type: 'boolean', alias: 'e', desc: 'Only destroy requested stacks, don\'t include dependees' })
-      .option('force', { type: 'boolean', alias: 'f', desc: 'Do not ask for confirmation before destroying the stacks' }))
+      .option('force', { type: 'boolean', alias: 'f', desc: 'Do not ask for confirmation before destroying the stacks' })
+      .option('exit-on-config-complete', { type: 'boolean', desc: 'Whether to exit on configuration complete', default: false }))
```

Comment:
Is this being ignored?  I only see support for `deploy`? And if this isn't being ignored, can we add an integration test for `destroy` case?

Reply:
This feature does not support destroy stacks so I removed all relevant code about the `destroy` case.